cannot find './helper/debug'

When I try running it via npx I get the following output:

```
npx: installed 44 in 1.53s
Cannot find module './helper/debug'
Require stack:
- C:\Users\Nikolaj\AppData\Roaming\npm-cache\_npx\18536\node_modules\es5-validator\index.js
```

--- a/package.json
+++ b/package.json
@@ -4,7 +4,7 @@
   "description": "Validate JS source code against ES5 to avoid breaking you App by shipping ES6+ code to production silently",
   "main": "index.js",
   "files": [
-    "fetch.js",
+    "helper",
     "index.js"
   ],
   "scripts": {

